### PR TITLE
Kotlin extensions

### PR DESCRIPTION
**Pull request by martin371998**
Used Built-In Kotlin Extension Functions for simple checks

### CODE REVIEW

File: app/src/main/java/com/example/euriskocodechallenge/ui/home/news/NewsFragment.kt [hunk 1]
```diff
@@ -79,7 +79,7 @@ class NewsFragment : Fragment() {
     private fun observeViewModel(view: View) {
         //Observe Retrofit Result -> Set Recycler View Data, Dismiss Progress Bar
         viewModel.response.observe(viewLifecycleOwner) {
-            if (it != null) {
+            if (!it.equals(null)) {
```
Comment (joe-abouassi) [inline app/src/main/java/com/example/euriskocodechallenge/ui/home/news/NewsFragment.kt]: @martin371998 avoid null checks, instead use safe calls and ?.let { }

File: app/src/main/java/com/example/euriskocodechallenge/ui/home/viewmodel/MoreViewModel.kt [hunk 2]
```diff
@@ -31,17 +31,9 @@ class MoreViewModel @Inject constructor(
...
-                            fName,
-                            lName,
-                            it.password,
-                            imageBitmap
-                        )
-                        userDatabaseRepository.updateUser(user)
+                userDatabaseRepository.getUserById(id).collectLatest { user ->
+                    user?.let {
+                        userDatabaseRepository.updateUser(it)
                         Log.d(Constants.TAG, "Updated User")
```
Comment (joe-abouassi) [inline app/src/main/java/com/example/euriskocodechallenge/ui/home/viewmodel/MoreViewModel.kt]: @martin371998 avoid pushing debug logs, use a logging function with debug enabled

**Review by joe-abouassi — CHANGES_REQUESTED** [review]

**Review by joe-abouassi — APPROVED** [review]

File: app/src/main/java/com/example/euriskocodechallenge/ui/home/news/NewsFragment.kt (hunk 1 above)
Comment (martin371998) [reply app/src/main/java/com/example/euriskocodechallenge/ui/home/news/NewsFragment.kt]: done in another branch

File: app/src/main/java/com/example/euriskocodechallenge/ui/home/viewmodel/MoreViewModel.kt (hunk 2 above)
Comment (martin371998) [reply app/src/main/java/com/example/euriskocodechallenge/ui/home/viewmodel/MoreViewModel.kt]: done in another branch

**Pull request merged**